File: lang/c++/test/fun/vac/util/TestRunnerTest.cpp
#include <fun/vac/util/TestRunner.hpp>

using namespace std;

void testParseTest() {
    for (int i = 0; i < 10; i++) {
        if ((i & 1) == 0) {
            TestRunner::parseTest(false);
        } else {
            TestRunner::parseTest(true);
        }
    }
}

int main() {
    testParseTest();
}

File: lang/php/test/fun/vac/util/TestRunnerTest.php
<?php
    require_once "fun/vac/util/TestRunner.php";

    use fun\vac\util\TestRunner;

    function testParseTest(): void {
        for ($i = 0; $i < 10; $i++) {
            TestRunner\parseTest(($i & 1) != 0);
        }
    }

    if (count(debug_backtrace()) == 0) {
        testParseTest();
    }
?>
